tests/unit/docker_images/test_validator_harness.py

```diff
@@ -35,7 +35,13 @@ def setUp(self):
         s3obj.put(Body=self.file_contents)
         self.s3_url = f"s3://{self.upload_config.bucket_name}/{self.s3_object_key}"
 
+        # A file will be created while the subprocess is running under the name SUBPROCESS_LOG_FILE. This will have
+        # to be cleaned up in tearDown once each test has run.
+        self.log_filename = "SUBPROCESS_LOG_FILE"
```

Comment:
The log_filename should probably include something unique about the request? Many validators will be running on a single machine